brave/brave-core
# Publishers can now be included and excluded

Resolves https://github.com/brave/brave-browser/issues/1283

Ledger native https://github.com/brave-intl/bat-native-ledger/pull/144

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:

1. Start Brave
2. Add publishers to rewards auto-contribution
3. Visit an added publisher and open Rewards panel
4. Click 'Include in auto-contribute'
5. Check publisher_info_db and ensure that excluded was changed to 1
6. Click 'Include in auto-contribute' in panel for same site
7. Check publisher_info_db and ensure that excluded was changed to 2

~Toggle switch currently does not slide and will be handled in future PR.
(Based on native and script type mismatches)~


## Reviewer Checklist:
